specs/notation-cli.md

```diff
@@ -0,0 +1,36 @@
+# Notation CLI
+
+The following spec outlines the notation CLI.
+The CLI commands are what's currently available in [notation v0.7.1-alpha.1](https://github.com/notaryproject/notation/releases/tag/v0.7.1-alpha.1)
+
+
+```bash
+notation help
+NAME:
+   notation - Notation - Notary V2
+
+USAGE:
+   notation [global options] command [command options] [arguments...]
+
+VERSION:
+   0.0.0-SNAPSHOT-17c7607
+
+AUTHOR:
+   CNCF Notary Project
+
+COMMANDS:
+   sign               Signs artifacts
+   verify             Verifies OCI Artifacts
+   push               Push signature to remote
+   pull               Pull signatures from remote
+   list, ls           List signatures from remote
+   certificate, cert  Manage certificates used for verification
```

Comment:
Given that we decided to manage certificates using the trust policy defnition, should we drop 
`certificate` and `key`

Reply:
Also considering this usecase for local file with passpharse should we possibly have `--passpharse` as an option for sign. Are we planning to include sub command options for the CLI here? 
https://github.com/notaryproject/roadmap/issues/46

Reply:
We need to think through the initial experience and production experiences here.
What is the user experience to get started?
What is the user experience to add remote signed keys?
What is the user experience to add certs for verification? 

Today, this can be accomplished with some straight-forward commands. 
What will this experience look like in RC1?

Reply:
`notation cert add` is an alpha 1 command created before [trust store and trust policy](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md) spec was finalized, and is not compatible with the spec. RC1 implements spec which uses trust store and trust policy instead of a global certificate list in `config.json`. This `cert add` command needs rename/changes along with possibly new commands to manage trust policy, which are currently not in scope for for RC1 release. RC1 is prioritizing core functionality, `sign` & `verify` commands, over usability of other features. We may update some other commands like `key add` or `plugin list` as part of ongoing work, but these are not blockers for RC1 release. Users can accomplish these scenarios by manually editing trust store, trust policy, config.json/signingkeys.json . We plan to add/update commands post RC1 and before GA release.

See examples here for how this needs to be configured for RC1 - [trust store](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-store) , [trust policy](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy-schema).